modules/ocf_puppetdb/manifests/init.pp

```diff
@@ -1,3 +1,31 @@
 class ocf_puppetdb {
-  package { ['puppetdb', 'puppetdb-termini']:; }
+  include ocf_ssl::default_bundle
```

Comment:
I originally added this for [puppetboard](https://github.com/voxpupuli/puppetboard), but I think putting it on Marathon is cleaner, so I'll do that and remove this line and the certs.

Reply:
Never mind, I'll probably just proxy this to puppetboard, like with the web IRC.

Reply:
So proxy it from the puppetdb server so you can visit it at puppetdb.ocf.berkeley.edu? Seems convenient.

I seem to recall you said that puppetboard required authentication, presumably because it lets you change things/view secrets. If so, would the idea be to put nginx here to do SSL and proxy to the LB, and then each puppetboard instance would have another nginx which does authentication?

Reply:
I'm not exactly sure, but I do think that having it be available at `puppetdb.ocf.berkeley.edu` would be nice. Might even be nice to be available at `puppet.ocf.berkeley.edu`, but that might be weird if we implement something else that wants to run a web server on lightning, and then we wouldn't want it to be at both `puppet.` and `puppetdb.`, because that would be redundant and a bit confusing. Currently I'm leaning towards just putting it at `puppetdb.` because that's more indicative of what it does, but I'm open to suggestions on which of those would be better.

As for authentication, I think having the inner nginx do the authentication would be the most secure and make the most sense, like RT does it, except simpler because it doesn't have to pass through any credentials or anything and can use nginx. Just having this instance do SSL termination and proxying like the [web IRC proxy](https://github.com/ocf/puppet/blob/master/modules/ocf_irc/manifests/webirc.pp) does seems easy enough.